LORIS_react.README.md

```diff
@@ -43,35 +43,35 @@ Download the NodeJS package from [here](https://nodejs.org/en/)
 
 ####1.1.2 NPM
 
-Once NodeJS is installed use the following to instal NPM
+Once NodeJS is installed, use the following to install NPM
 
 ```
 curl -L https://www.npmjs.com/install.sh | sudo sh
 ```
 
 ###1.2 Installing Grunt
 
-Once NodeJS and NPM are installed you are ready to install Grunt. First in your terminal ```cd``` into the LORIS home directory. 
-Once there run the following command:
+Once NodeJS and NPM are installed, you are ready to install Grunt. First, ```cd``` into the LORIS home directory in your terminal. 
```

Comment:
I know they're already there, but why triple backticks instead of single? Triple is usually for multi-line code blocks, single is for inline code/references.